### PR TITLE
[6.0] The admin model does return a stdClass instead of CMSObject

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
Followup of https://github.com/joomla/joomla-cms/pull/42717, to return a `\stdClass` instead of `CMSObject` in the Adminmodel getItem function. Developers should work directly on the properties on the returned item and not call the deprecated set or get function.

### Testing Instructions
- Install 3rd party content plugins
- Create an article
- Open the article on the front end

### Actual result BEFORE applying this Pull Request
All works.

### Expected result AFTER applying this Pull Request
All works.

### Link to documentations
Please select:
- [x] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/237
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: configuration.php.bak [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?php
```
Comment (Quy) [inline configuration.php.bak]: Exclude file?

File: libraries/src/MVC/Model/AdminModel.php [hunk 2]
```diff
@@ -1003,9 +1002,9 @@ public function getItem($pk = null)
             }
         }
 
-        // Convert to the CMSObject before adding other data.
-        $properties = $table->getProperties(1);
-        $item       = ArrayHelper::toObject($properties, CMSObject::class);
+        // Convert to \stdClass before adding other data
+        $properties = get_object_vars($table);
+        $item       = ArrayHelper::toObject($properties, \stdClass::class);
```
Comment (HLeithner) [inline libraries/src/MVC/Model/AdminModel.php]: \stdClass can be removed it's the default value

Comment (laoneo) [reply libraries/src/MVC/Model/AdminModel.php]: done

**Comment by Bodge-IT** [comment]
@laoneo what's needed to get these checks cleared?

**Comment by Bodge-IT** [comment]
I have tested this item :white_check_mark: successfully on 1b7bbd70a6f151760b11e89b89c3d6c427431081<br /><br />Worked in both states<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42961">issues.joomla.org/tracker/joomla-cms/42961</a>.</sub>

**Comment by laoneo** [comment]
Fixed the deprecated bug in the task model.

**Comment by Bodge-IT** [comment]
I have tested this item :white_check_mark: successfully on d1a1bbaf62ff48885112bc1a454ad0101351d564<br /><br />Content plugin worked  on unpatched and patched states<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42961">issues.joomla.org/tracker/joomla-cms/42961</a>.</sub>

**Comment by softforge** [comment]
I have tested this item :white_check_mark: successfully on d1a1bbaf62ff48885112bc1a454ad0101351d564<br /><br />I have tested this successfully but we will need an entry in the manual once 6.0 has been added to the manual<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42961">issues.joomla.org/tracker/joomla-cms/42961</a>.</sub>

**Comment by laoneo** [comment]
Made a pr already https://github.com/joomla/Manual/pull/237.

**Pull request merged**